pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -241,7 +241,7 @@ func (w *logWriter) write(msg *logMessage) error {
 	}
 	line := msg.log
 	if w.opts.timestamp {
-		prefix := append([]byte(msg.timestamp.Format(timeFormatOut)), delimiter[0])
+		prefix := append([]byte(msg.timestamp.UTC().Format(timeFormatOut)), delimiter[0])
```

Comment:
if other log messages use local time, this probably should remain local time as well